COMMON/HOST.xml

```diff
@@ -0,0 +1 @@
+<macro name="HOST" pattern="^([a-zA-Z0-9_-]+|[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+)(:[0-9]+)*$" description="Host name or IP address, optionally including :PORTNUMBER"  hasDefault="NO"/>
```

Comment:
Do we need both a `HOST` and `HOST_WITH_PORT` option? Not every place we ask for a HOST also allows a port. Also should that be `(:[0-9]+)?` rather than `(:[0-9]+)*` ?